src/viadot/orchestration/prefect/flows/vid_club_to_adls.py

```diff
@@ -0,0 +1,97 @@
+"""Download data from Vid CLub API and load it into Azure Data Lake Storage."""
+
+from typing import Any, Dict, List, Literal
+from prefect import flow
+from prefect.task_runners import ConcurrentTaskRunner
+from viadot.orchestration.prefect.tasks import df_to_adls, vid_club_to_df
+
+
+@flow(
+    name="Vid CLub extraction to ADLS",
+    description="Extract data from Vid CLub and load it into Azure Data Lake Storage.",
+    retries=1,
+    retry_delay_seconds=60,
+    task_runner=ConcurrentTaskRunner,
+)
+def vid_club_to_adls(
+    *args: List[Any],
+    source: Literal["jobs", "product", "company", "survey"] = None,
+    from_date: str = "2022-03-22",
+    to_date: str = None,
+    items_per_page: int = 100,
+    region: Literal["bg", "hu", "hr", "pl", "ro", "si", "all"] = None,
+    days_interval: int = 30,
+    cols_to_drop: List[str] = None,
+    vid_club_credentials: Dict[str, Any] = None,
```

Comment:
Do not expose credentials in flows/tasks. See the Best Practices section [here](https://viadot.docs.dyvenia.com/developer_guide/creating_a_prefect_flow/#creating-a-task)

Reply:
[e28d9ab](https://github.com/dyvenia/viadot/pull/1044/commits/e28d9abf8680da8341cb5413d93d61e369f68eb8)